NotebookAnalyzers/utils.py

```diff
@@ -27,19 +27,69 @@ def notebookToCode(filename):
     return py_code
 
 def functionsNumber(code):
-    """The function takes a python code and returns the number of function definitions"""
+    """The function takes a python code string and returns the number of function definitions"""
     tree = ast.parse(code)
     f_num=sum(isinstance(exp, ast.FunctionDef) for exp in tree.body)
     return f_num
 
 def notExecutedCells(notebook):
-    """The function takes the notebook dictionary and returns both the number empty cells in first position and the number of not executed cells in second position"""
-    empty_cells=0
+    """The function takes a dict representing a notebook and returns the number of not executed cells"""
     not_exec_cells=0
+    for cell in notebook["cells"]:
+        if cell["cell_type"] == 'code':
+            if cell['execution_count']==None and cell['source']!=[]:
+                not_exec_cells=not_exec_cells+1 #This is a not executed Python Cell containing actual code
+    return not_exec_cells
+
+def emptyCells(notebook):
+    """The function takes a dict representing a notebook and returns the number of empty cells"""
+    empty_cells=0
     for cell in notebook["cells"]:
         if cell["cell_type"] == 'code':
             if cell['execution_count']==None and cell['source']==[]:
                 empty_cells=empty_cells+1 #This is an empty Python Cell
-            elif cell['execution_count']==None and cell['source']!=[]:
-                    not_exec_cells=not_exec_cells+1 #This is a not executed Python Cell containing actual code
-    return empty_cells,not_exec_cells
+    return empty_cells
+
+def importsCorrectPosition(code):
```

Comment:
We do I find this and the following function definitions here? From the PR title, I assumed that I had to review only the functions `notExecutedCells` and `emptyCells`.

Reply:
Trovi la definizione di funzioni presenti sul master in quanto ho dovuto risolvere i conflitti fra questo branch e il master per poter rendere il branch mergiabile. Sia il master che questo branch lavorano sullo stesso file utils.py ma il master contiene le funzioni degli altri branch che sono stati mergiati, branch che hai già visionato e di cui hai approvato il merge. Questo è successo in quanto ho dovuto lavorare con più branch sullo stesso file. Non so se esiste una soluzione migliore, forse non risolvere i conflitti prima della tua approvazione della pull request può essere una soluzione

Reply:
Sulle altre pull request non ti sei accorto di ciò appunto perchè prima di risolvere i conflitti ho atteso la tua revisione delle funzioni interessate sul branch che stavi visionando e solo dopo ho risolto i conflitti, indicando di lasciare sia le funzioni del branch specifico, sia quelle già presenti sul master